packages/protocol/scripts/bash/contract-exclusion-regex.sh

```diff
@@ -0,0 +1,14 @@
+# Exclude test contracts, mock contracts, contract interfaces, Proxy contracts, inlined libraries,
+# MultiSig contracts, and the ReleaseGold contract.
+
+
+
+CONTRACT_EXCLUSION_REGEX=".*Test|Mock.*|I[A-Z].*|.*Proxy|MultiSig.*|ReleaseGold|SlasherUtil|UsingPrecompiles"
+
+# Before CR7, they had been deployed, so don't rebuild the report
+VERSION_NUMBER=$(echo "$OLD_BRANCH" | tr -dc '0-9')
+
+if [ $VERSION_NUMBER -gt 6 ]
+  then
+  CONTRACT_EXCLUSION_REGEX="$CONTRACT_EXCLUSION_REGEX|^UsingRegistry"
```

Comment:
are there any risks in excluding UsingRegistry from comparisons? or is this absolutely required?

Reply:
UsingRegistry shouldn't have been deployed in the first place either, as they were supposed to be mixin. Deploying it looks like a bug in the tooling all along.